### DISCUSSION TITLE
Can't deploy because of @vercel/og ImageResponse  error

### DISCUSSION THREAD

**Discussion by alex-robert-dev** [vercel/community#3029] [Bug Report]
### Page to Investigate

https://www.fond.ch

### Not working:
- Vercel Node.js Version: 18.x
- "next": "^13.4.7",
- "@vercel/og": "^0.5.8"

Error Messages:

- "Attempted import error: 'ImageResponse' is not exported from '@vercel/og' (imported as 'ImageResponse')."
- Error: The Edge Function "api/og" is referencing unsupported modules:
--
09:08:02.643 | - index.js: ./resvg.wasm?module, ./yoga.wasm?module, url
09:08:04.816 | NOW_SANDBOX_WORKER_EDGE_FUNCTION_UNSUPPORTED_MODULES: The Edge Function "api/og" is referencing unsupported modules:	- index.js: ./resvg.wasm?module, ./yoga.wasm?module, url

### Working:
- Vercel Node.js Version: 18.x
- "next": "^13.2.4",
- "@vercel/og": "^0.5.0"

Didn't change any code between the versions

**Comment by amyegan** [comment]
Hi @alex-robert-dev. I tried deploying a project with Next.js and `@vercel/og`, but I wasn't blocked by an error. If you can share your public repo or a minimal reproduction, that will let me get a better look at how this error happens.

**Reply by thereisryan** [reply]
Encountered the same error following the docs for pages/api implementation :(
